src/bitmap/inherent.rs

```diff
@@ -43,6 +43,41 @@ impl RoaringBitmap {
         container.insert(index)
     }
 
+    /// Adds a value to the set.
+    /// The value **must** be strictly bigger than the maximum value in the set.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// use roaring::RoaringBitmap;
+    ///
+    /// let mut rb = RoaringBitmap::new();
+    /// rb.push(1);
+    /// rb.push(3);
+    /// rb.push(5);
+    ///
+    /// assert_eq!(rb.iter().collect::<Vec<u32>>(), vec![1, 3, 5]);
+    /// ```
+    pub fn push(&mut self, value: u32) {
```

Comment:
```suggestion
    pub fn push_back(&mut self, value: u32) {
```

I am just proposing that as it is a little bit more explicit/obvious, what do you think?

Reply:
I used push because to be consistent with Vectors, but If you think this is better go for it!

Reply:
Not sure it is better, can the pushed integer be also equal to the last integer in the collection, too?

Reply:
Thank you for your note, you are right, I have just fixed that case in the [last commit](https://github.com/Nemo157/roaring-rs/pull/70/commits/d9f0de7f71e8e090648111ed8b3b7d4f64ded610)